html5/js/OffscreenDecodeWorker.js

```diff
@@ -527,21 +253,16 @@ onmessage = function (e) {
       const wid = packet[1];
       wd = offscreen_canvas.get(wid);
       if (wd) {
-        wd.decode_draw_packet(packet);
+        wd.queue_draw_packet(packet);
       } else {
-        send_decode_error(
-          packet,
-          `no window decoder found for wid ${wid}, only:${[
-            ...offscreen_canvas.keys(),
-          ].join(",")}`
-        );
+        send_decode_error(packet, `no window decoder found for wid ${wid}, only:${[...offscreen_canvas.keys(),].join(",")}`);
       }
       break;
     }
     case "redraw":
       wd = offscreen_canvas.get(data.wid);
       if (wd) {
-        wd.redraw();
+        //wd.redraw();
```

Comment:
I haven't checked this yet.
If this is no longer needed, let's rip it out.
If it is... then we have to ask the client to send a full window refresh since we don't keep any decoded pixels around?

Reply:
I did not find a case where this is needed. Could you test this?
Would be nice to not need it at all.

Reply:
See https://github.com/Xpra-org/xpra-html5/issues/144#issuecomment-1026963759

Reply:
Thanks. I could not reproduce the issue. Also convertToBlob() & transferToImageBitmap() seems to make the canvas a bit slower. I dont want that so I ended up painting to another offscreencanvas named 'still' and flipping buffers on 'redraw'.
What do you think about that?

Reply:
That works but it's still (!) going to be expensive.
I'll play with it and see if we can get rid of the whole thing.